LoopKitUI/Views/CorrectionRangeScheduleEditor.swift

```diff
@@ -0,0 +1,177 @@
+        self.unit = unit
+        self.minValue = minValue
+        self.save = save
+    }
+
+    public var body: some View {
+        ScheduleEditor(
+            title: Text("Correction Ranges", comment: "Title of correction range schedule editor"),
+            description: description,
+            scheduleItems: $scheduleItems,
+            initialScheduleItems: initialSchedule?.items ?? [],
+            defaultFirstScheduleItemValue: defaultFirstScheduleItemValue,
+            valueContent: { range, isEditing in
+                GuardrailConstrainedQuantityRangeView(range: range.quantityRange(for: self.unit), unit: self.unit, guardrail: self.guardrail, isEditing: isEditing)
+            },
+            valuePicker: { scheduleItem, availableWidth in
+                GlucoseRangePicker(
+                    range: Binding(
+                        get: { scheduleItem.wrappedValue.value.quantityRange(for: self.unit) },
+                        set: { quantityRange in
+                            withAnimation {
+                                scheduleItem.wrappedValue.value = quantityRange.doubleRange(for: self.unit)
+                            }
+                        }
+                    ),
+                    unit: self.unit,
+                    minValue: self.minValue,
+                    guardrail: self.guardrail,
+                    stride: self.stride,
+                    availableWidth: availableWidth
+                )
+            },
+            actionAreaContent: {
+                guardrailWarningIfNecessary
+            },
+            onSave: { scheduleItems in
+                if self.crossedThresholds.isEmpty {
+                    self.saveAndDismiss()
+                } else {
+                    self.showingConfirmationAlert = true
+                }
+            }
+        )
+        .alert(isPresented: $showingConfirmationAlert, content: confirmationAlert)
+    }
+
+    private var stride: HKQuantity {
+        switch unit {
+        case .milligramsPerDeciliter:
+            return HKQuantity(unit: .milligramsPerDeciliter, doubleValue: 1)
+        case .millimolesPerLiter:
+            return HKQuantity(unit: .millimolesPerLiter, doubleValue: 0.1)
+        default:
+            fatalError("Unsupported glucose unit \(unit)")
+        }
+    }
+
+    var defaultFirstScheduleItemValue: DoubleRange {
+        switch unit {
+        case .milligramsPerDeciliter:
+            return DoubleRange(minValue: 100, maxValue: 120)
+        case .millimolesPerLiter:
+            return DoubleRange(minValue: 5.6, maxValue: 6.7)
+        default:
+            fatalError("Unsupposed glucose unit \(unit)")
+        }
+    }
+
+    var description: Text {
+        Text("The correction range is the glucose range that you would like the App to correct your glucose to by adjusting insulin dosing.", comment: "Description of correction range setting")
+    }
```

Comment:
This the current language from Melissa Lee and Matt's doc, but I think it could read more clearly. Discussing it with them now; I don't expect this version to be final.